addons/skin.estuary/xml/Includes.xml

```diff
@@ -1696,7 +1696,7 @@
 	</include>
 	<include name="MediaInfoListLayout">
 		<param name="height">125</param>
-		<param name="width">880</param>
+		<param name="width">940</param>
```

Comment:
Changing this has affected the video manager dialog list width - it now overlaps the buttons.

Reply:
Yes it was an oversight not to adjust the video version/extra manager to reflect this change.

Reply:
Should be fixed now